review04/passenger-service/src/test/java/cz/muni/fi/airportmanager/passengerservice/repository/PassengerRepositoryTest.java

```diff
@@ -0,0 +1,163 @@
+import io.quarkus.test.TestReactiveTransaction;
+import io.quarkus.test.junit.QuarkusTest;
+import io.quarkus.test.vertx.UniAsserter;
+import jakarta.inject.Inject;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+@QuarkusTest
+class PassengerRepositoryTest {
+
+    @Inject
+    PassengerRepository passengerRepository;
+
+
+    @Test
+    @TestReactiveTransaction
+    void shouldFindNotificationsForPassenger(UniAsserter asserter) {
+        Passenger passenger = createTestPassenger();
+        Notification notification = createTestNotification();
+
+        asserter.execute(() -> passengerRepository.deleteAll())
+                .execute(() -> passengerRepository.persist(passenger))
+                .execute(() -> {
+                    passenger.addNotification(notification);
+                    return passengerRepository.persist(passenger);
+                })
+                .assertThat(
+                        () -> passengerRepository.findNotificationsForPassenger(passenger.getId()),
+                        notifications -> {
+                            assertEquals(1, notifications.size());
+                            assertEquals(notification.message, notifications.get(0).message);
+                        }
+                );
+    }
+
+    @Test
+    @TestReactiveTransaction
+    void shouldAddNotificationByFlightId(UniAsserter asserter) {
+        Passenger passenger = createTestPassenger();
+        Notification notification = createTestNotification();
+
+        asserter.execute(() -> passengerRepository.deleteAll())
+                .execute(() -> passengerRepository.persist(passenger))
+                .execute(() -> passengerRepository.addNotificationByFlightId(passenger.getFlightId(), notification))
+                .assertThat(
+                        () -> passengerRepository.findNotificationsForPassenger(passenger.getId()),
+                        notifications -> assertTrue(notifications.stream().anyMatch(n -> n.message.equals(notification.message)))
+                );
+    }
+
+    @Test
+    @TestReactiveTransaction
+    void shouldFindPassengersForFlight(UniAsserter asserter) {
+        Passenger passenger = createTestPassenger();
+
+        asserter.execute(() -> passengerRepository.deleteAll())
+                .execute(() -> passengerRepository.persist(passenger))
+                .assertThat(
+                        () -> passengerRepository.findPassengersForFlight(passenger.getFlightId()),
+                        passengers -> {
+                            assertEquals(1, passengers.size());
+                            assertEquals(passenger.getId(), passengers.get(0).getId());
+                        }
+                );
+    }
+
+    @Test
+    @TestReactiveTransaction
+    void shouldFindHydratedNotifications(UniAsserter asserter) {
```

Comment:
As I already commented before, what is hydrated notification???

Reply:
I forgot to change the name of the test as well. Will fix.